src/platform-includes/session-replay/setup/javascript.capacitor.mdx

```diff
@@ -1,6 +1,8 @@
 Several options are supported and passable using the integration constructor. See the [configuration documentation](/platforms/javascript/guides/capacitor/session-replay/configuration/) for more details. To set up the integration, add the following to your Sentry initialization:
 
-<Note>The minimal supported version of Sentry's SDK for Capacitor is 0.11.1.</Note>
+<Note>
+  The minimal supported version of Sentry's SDK for Capacitor is 0.11.1.
+</Note>
```

Comment:
whoa new note formatting dropped. I feel like I've asked this before, but will it mess up our spacing for the notes that contain markdown & require newlines?

Reply:
So if your note contains markdown, when you put the newlines before and after the prose then remark will treat it as markdown. Without the newlines then it is treated fully as a react javascript component (as is the case here), in which case no markdown formatting will be applied.

Reply:
makes sense, thanks!